### PR TITLE
{bio}[dummy/] IMOD v4.7.15

### PR DESCRIPTION
**Pull request by pescobar**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/i/IMOD/IMOD-4.7.15_RHEL6-64_CUDA6.0.eb [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+# It's not possible to define a function in the module file 
+modloadmsg = """Bash users run: 'source $EBROOTIMOD/IMOD-linux.sh'
+csh users run: 'source $EBROOTIMOD/IMOD-linux.csh'"""
+
+modextravars = {
+    'IMOD_JAVADIR': '\$EBROOTJAVA',
+}
+
+sanity_check_paths = {
+    'files': ['IMOD/bin/imod'],
```
Comment (boegel) [inline easybuild/easyconfigs/i/IMOD/IMOD-4.7.15_RHEL6-64_CUDA6.0.eb]: `imod` will not be available via `$PATH` this way...

we should add `IMOD/bin` to `$PATH` via `modextrapaths`?

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2006.delcatty.os - Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/8aea0a2242102f9cfd7324fe9c9926d4 for a full test report.

File: easybuild/easyconfigs/i/IMOD/IMOD-4.7.15_RHEL6-64_CUDA6.0.eb (hunk 1 above)
Comment (pescobar) [inline easybuild/easyconfigs/i/IMOD/IMOD-4.7.15_RHEL6-64_CUDA6.0.eb]: the easyconfig includes this 
```modloadmsg = """Bash users run: 'source $EBROOTIMOD/IMOD-linux.sh'
csh users run: 'source $EBROOTIMOD/IMOD-linux.csh'"""```

when source this file the PATH and other required variables will be defined

Comment (boegel) [inline easybuild/easyconfigs/i/IMOD/IMOD-4.7.15_RHEL6-64_CUDA6.0.eb]: OK, missed that, thanks @pescobar

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2159.delcatty.os - Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/b5531637834ea69ec275b9529d7f6f15 for a full test report.

**Comment by boegel** [comment]
lgtm

**Comment by boegel** [comment]
Going in, thanks @pescobar!

**Pull request merged**